packages/camera/camera_web/lib/src/camera_web.dart

```diff
@@ -324,22 +324,69 @@ class CameraPlugin extends CameraPlatform {
 
   @override
   Stream<DeviceOrientationChangedEvent> onDeviceOrientationChanged() {
-    throw UnimplementedError(
-      'onDeviceOrientationChanged() is not implemented.',
-    );
+    final orientation = window?.screen?.orientation;
```

Comment:
I'm surprised these are not breaking analysis, we used to require to specify types in these definitions.